guide/how-it-works/sign-in-with-bitcoin.md

```diff
@@ -0,0 +1,161 @@
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/bridged-connection.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/bridged-connection@2x.png"
+   alt-text = "Communication flow between a browser window and a smart phone wallet"
+   width = 800
+   height = 411
+%}
+
+This does not necessarily have to be done via QR code. The most basic method could even involve manually copying and pasting messages and identifiers back and forth.
+
+## Signing in
+
+The biggest hurdle at this stage is that most users will not be familiar with this authentication method and are not likely to have a wallet they can readily use. Ensure that there are concise explanations and tips to get users started.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-info.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-info@2x.png"
+   alt-text = "Sign in screen with tips for wallets to use and general information"
+   width = 800
+   height = 411
+%}
+
+Authentication is a simple process in which the service sends a message to the wallet. The wallet then uses cryptography to generate a unique identifier from the message, which it sends back to the service. This identifier can then be used to establish an account.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-request.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/signin-request@2x.png"
+   alt-text = "Browser window with a modal for approving web service authentication"
+   width = 800
+   height = 411
+%}
+
+Keep in mind that your service will not be able to retrieve user names, emails, balances, transactions, or other information from the wallet. Only the minimal information to identify the user is communicated. If you need to communicate with users about their accounts or activity, make sure you request an email or other contact point.
+
+## Signing out
+
+Ensuring users are signed out (manually or via session time-outs) is important. The account may have payment allowances, or other problematic permissions, that could otherwise be abused by third parties.
+
+## Making payments
+
+Typically, your service will prompt the user for a payment when the user tries to interact with the service. Show the most appropriate option right away based on how the user has previously used your service (direct or bridged, [see above]({{ '#wallet-service-communication' | relative_url }})).
```

Comment:
```suggestion
Typically, a service will prompt the user for a payment when the user tries to interact with the service. Show the most appropriate option right away based on how the user has previously used your service (direct or bridged, [see above]({{ '#wallet-service-communication' | relative_url }})).
```

How would a payment different between using direct or bridges? A visual for this would be handy. 

Same point as above, makes more sense for a how it works to talk like this.

Reply:
Added an image for this.